.github/workflows/check-samconfig.yml

```diff
@@ -0,0 +1,14 @@
+name: "Ensure all app templates have a sample samconfig.toml file"
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  samconfig-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout commit
+        uses: actions/checkout@v3
+      - name: Run checking script
+        run: python scripts/check_samconfig_exists.py
```

Comment:
What happens if this script fails? Does it block all merges? Do we need tests for it? Not blocking approval since the actions have passed.

Reply:
AFAIK, it can only be added to the required approval flow in the PR settings after merging it in. Until then, it'll show up as a failure but shouldn't block any PRs from being merged.